packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -0,0 +1,179 @@
+    // loose generates more ES5-like code but does not strictly adhere to the ES2015 spec (Typescript is loose)
+    // 'transform-typeof-symbol' generates slower code
+    presets: [
+      ['@babel/preset-env', { modules: false, loose: true, exclude: ['transform-typeof-symbol'] }],
+    ],
+    minified: true,
+    sourceMaps: options.sourceMaps,
+  });
+
+  const newFilePath = options.filename.replace('es2015', 'es5');
+
+  // Adjust lazy loaded scripts to point to the proper variant
+  if (options.runtime) {
+    code = code.replace('"-es2015.', '   "-es5.');
+  }
+
+  if (options.sourceMaps && manualSourceMaps && options.map) {
+    const generator = new SourceMapGenerator();
+    let sourceRoot;
+    await SourceMapConsumer.with(options.map, null, originalConsumer => {
+      sourceRoot = 'sourceRoot' in originalConsumer ? originalConsumer.sourceRoot : undefined;
+
+      return SourceMapConsumer.with(map, null, newConsumer => {
+        newConsumer.eachMapping(mapping => {
+          if (mapping.originalLine === null) {
+            return;
+          }
+
+          const originalPosition = originalConsumer.originalPositionFor({
+            line: mapping.originalLine,
+            column: mapping.originalColumn,
+          });
+
+          if (
+            originalPosition.line === null ||
+            originalPosition.column === null ||
+            originalPosition.source === null
+          ) {
+            return;
+          }
+
+          generator.addMapping({
+            generated: {
+              line: mapping.generatedLine,
+              column: mapping.generatedColumn,
+            },
+            name: originalPosition.name || undefined,
+            original: {
+              line: originalPosition.line,
+              column: originalPosition.column,
+            },
+            source: originalPosition.source,
+          });
+        });
+      });
+    });
+
+    map = generator.toJSON();
+    map.file = path.basename(newFilePath);
+    map.sourceRoot = sourceRoot;
+  }
+
+  if (options.optimize) {
+    // Note: Investigate converting the AST instead of re-parsing
+    // estree -> terser is already supported; need babel -> estree/terser
+
+    // Mangle downlevel code
+    const result = minify(code, {
+      compress: false,
+      mangle: true,
+      safari10: true,
+      toplevel: true,
+      output: {
+        ascii_only: true,
+        ecma: 5,
+        webkit: true,
+      },
+      sourceMap: options.sourceMaps && {
+        filename: path.basename(newFilePath),
+        content: map,
+      },
+    });
+
+    if (result.error) {
+      return Promise.reject(result.error);
+    }
+
+    code = result.code;
+    map = result.map;
+
+    // Mangle original code
+    await mangleOriginal(options);
+  } else if (map) {
+    map = JSON.stringify(map);
+  }
+
+  if (map) {
+    code += `\n//# sourceMappingURL=${path.basename(newFilePath)}.map`;
```

Comment:
This should not be appended if sourcemaps are hidden.

Reply:
Done.